frontends/ol-components/src/components/ThemeProvider/MITLearnGlobalStyles.tsx

```diff
@@ -3,26 +3,8 @@
 import React from "react"
 import { css, Global } from "@emotion/react"
 import { theme } from "./ThemeProvider"
-import { preload } from "react-dom"
-
-/**
-    Font files for Adobe neue haas grotesk.
-    WARNING: This is linked to chudzick@mit.edu's Adobe account.
-    We'd prefer a non-personal MIT account to be used.
-    See https://github.com/mitodl/hq/issues/4237 for more.
-
-    Ideally the font would be loaded via a <link /> tag; see
-      - https://nextjs.org/docs/app/api-reference/functions/generate-metadata#unsupported-metadata
-      - https://github.com/vercel/next.js/discussions/52877
-      - https://github.com/vercel/next.js/discussions/50796
```

Comment:
I'm not sure I was interpretting these docs correctly... They say:

> # Unsupported Metadata:
>
> The following metadata types do not currently have built-in support. **However, they can still be rendered in the layout or page itself.**

(emphasis mine).

Sticking the stylesheet inside a head tag definitely does work.